<launch>


    <arg name="detection_sub_topic" default="smart_tracking/obstacles" />
    <arg name="camera_sub_topic" default="image_raw" />
    <arg name="markers_pub_topic" default="markers" />
    <arg name="interpolated_pub_topic" default="interpolated_detections" />

    <arg name="online" default="false"/>
    <arg name="marker_visualization" default="true"/>



    <node pkg="didi_challenge" type="interpolation" name="interpolation" output="screen">
        <param name="detection_sub_topic" value="$(arg detection_sub_topic)"/>
        <param name="camera_sub_topic" value="$(arg camera_sub_topic)"/>
        <param name="markers_pub_topic" value="$(arg markers_pub_topic)"/>
        <param name="interpolated_pub_topic" value="$(arg interpolated_pub_topic)"/>
        <param name="online" value="$(arg online)"/>
        <param name="marker_visualization" value="$(arg marker_visualization)"/>

</node>

</launch>
